crates/bevy_ecs/src/system/commands.rs

```diff
@@ -178,6 +180,13 @@ impl<'a> Commands<'a> {
         });
     }
 
+    /// Run a one-off [`System`].
+    pub fn run_system(&mut self, system: impl System<In = (), Out = ()>) {
```

Comment:
This is a bit of bikeshedding, but I feel the name `run_system` doesn't really express the intent of only running the system once. I could see someone easily getting confused thinking this adds the system to the overall app and expects is to execute each frame.
Perhaps we could use `run_once` or `one_shot`, `run_one_shot` or something to that extent.

Reply:
`one_shot` feels most natural to me, but I think it's not going to be clear to ESL users or those outside of gaming culture. 

`run_once` is probably best.

Reply:
I think it should still have `system` in the name:
* makes it clearer and parallel with `add_system`
* the command struct name should mirror the method on commands, and I think a `RunOnce` is very not clear from the name